models/labels/arbitrage_traders/labels_arbitrage_traders.sql

```diff
@@ -0,0 +1,3 @@
+{{config(alias='arbitrage_traders')}}
```

Comment:
you need to expose this view, you can use this instead of the current config statement:

`{{config(alias='arbitrage_traders',
       post_hook='{{ expose_spells(\'["ethereum"]\',
                                    "sector",
                                    "labels",
                                    \'["alexth"]\') }}')}}`